src/components/Contributors.tsx

```diff
@@ -59,17 +35,61 @@ const Contributors: React.FC<IProps> = () => {
         have contributed so far!
       </p>
 
-      <Container>
+      <Flex flexWrap="wrap">
         {contributorsList.map((contributor, idx) => (
-          <ContributorCard
-            key={idx}
-            image={contributor.avatar_url}
-            to={contributor.profile}
-            title={contributor.name}
-            alt={contributor.name}
-          />
+          <LinkBox
+            as="div"
+            maxWidth="132px"
+            margin="0.5rem"
+            boxShadow="0px 14px 66px rgba(0, 0, 0, 0.07), 0px 10px 17px rgba(0, 0, 0, 0.03), 0px 4px 7px rgba(0, 0, 0, 0.05)"
+            _hover={{
+              textDecoration: "none",
+              borderRadius: "4px",
+              boxShadow: "0px 8px 17px rgba(0, 0, 0, 0.15)",
+              background: "tableBackgroundHover",
+              transition: "transform 0.1s",
+              transform: "scale(1.02)",
+            }}
+            _focus={{
+              textDecoration: "none",
+              borderRadius: "4px",
+              boxShadow: "0px 8px 17px rgba(0, 0, 0, 0.15)",
+              background: "tableBackgroundHover",
+              transition: "transform 0.1s",
+              transform: "scale(1.02)",
+            }}
+          >
+            <Box display="flex" flexDirection="row" minHeight="100px">
```

Comment:
Are these styles props needed? we only have one child, correct? and the image has a fixed width/height.

Reply:
true, thanks